cgi-bin/ljlib.pl

```diff
@@ -242,30 +250,6 @@ sub gearman_client {
     return $client;
 }
 
-sub mogclient {
-    return $LJ::MogileFS if $LJ::MogileFS;
```

Comment:
Does it make more sense to do that now, or to keep it around as a way of quickly seeing which other places still need to be updated? Or are you planning to update them all before this is merged?